portals/package-lock.json

```diff
@@ -1,34 +1,12112 @@
 {
-  "name": "root",
+  "name": "@wso2apim/portals",
```

Comment:
There shouldn't be any package-lock changes if the PR is not introducing new libraries or updating them.